### PR TITLE
New package: IAMCloud.CloudDriveMapper version 2.20.0.1

### PR DESCRIPTION
**Pull request by kris6673**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/161819)

### CODE REVIEW

**Comment by kris6673** [comment]
This link is found directly from their official download site [here](https://resources.iamcloud.net/).

File: manifests/i/IAMCloud/CloudDriveMapper/2.20.0.1/IAMCloud.CloudDriveMapper.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PackageIdentifier: IAMCloud.CloudDriveMapper
+PackageVersion: 2.20.0.1
+InstallerLocale: en-GB
+InstallerType: msi
+Scope: machine
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{C40FC2AA-5A07-4816-A6EC-F2E21C0C09B4}'
```
Comment (Trenly) [inline manifests/i/IAMCloud/CloudDriveMapper/2.20.0.1/IAMCloud.CloudDriveMapper.installer.yaml:13]:
```suggestion
ProductCode: 'Cloud Drive Mapper 2.20.0.1'
```

File: manifests/i/IAMCloud/CloudDriveMapper/2.20.0.1/IAMCloud.CloudDriveMapper.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ReleaseDate: 2023-07-31
+AppsAndFeaturesEntries:
+- UpgradeCode: '{18936EF5-FD8C-4EA6-8507-648BA0B16498}'
```
Comment (Trenly) [inline manifests/i/IAMCloud/CloudDriveMapper/2.20.0.1/IAMCloud.CloudDriveMapper.installer.yaml:16]:
```suggestion
```

**Review by Trenly — CHANGES_REQUESTED** [review]
It seems this MSI doesn't write a standard Product Code 
```
--> Comparing ARP Entries

DisplayName        DisplayVersion Publisher                ProductCode                 Scope
-----------        -------------- ---------                -----------                 -----
Cloud Drive Mapper 2.20.0.1       IAM Technology Group Ltd Cloud Drive Mapper 2.20.0.1 Machine
```

**Comment by kris6673** [comment]
> It seems this MSI doesn't write a standard Product Code
> 
> ```
> --> Comparing ARP Entries
> 
> DisplayName        DisplayVersion Publisher                ProductCode                 Scope
> -----------        -------------- ---------                -----------                 -----
> Cloud Drive Mapper 2.20.0.1       IAM Technology Group Ltd Cloud Drive Mapper 2.20.0.1 Machine
> ```

It seems to write 2 entries under Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall that are both removed if the program is uninstalled. picture
However if the "Cloud Drive Mapper 2.20.0.1" one is removed, it's gone from the controlpanel list.

I tried the changes you suggested and seems to work fine.

**Comment by stephengillie** [comment]
Verify:
image
This is a tray-based app.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**